### ISSUE TITLE
Killing webpack-serve leaves an extra node http server running on Win10

### ISSUE THREAD

**Issue by cjmoran** [webpack-contrib/webpack-serve#37]
* Operating System: Win 10 x64
* Node Version: v9.6.1
* NPM Version: 5.6.0
* webpack Version: 4.0.1
* webpack-serve Version: 0.1.5

This issue is for a:

- [x] **bug**
- [ ] **feature** request
- [ ] **modification** request

##### CLI Command

```console
  $ webpack-serve ./webpack.config.js
```

##### webpack.config.js

https://gist.github.com/cjmoran/7d220cf0616b6ffcd96614ace23e3202

### Expected Behavior
Hitting Ctrl+C in my console while running webpack-serve will kill webpack-serve, including the node server it's running.

### Actual Behavior
Hitting Ctrl+C in my console kills webpack-serve and the terminal exits the program, but I can see one of its two Node.js processes stick around in task manager.

This means that next time I go to start the server, I'll be met with this error:

```console
✖ ｢serve｣:  { Error: listen EADDRINUSE 127.0.0.1:8080
    at Object._errnoException (util.js:1003:13)
    at _exceptionWithHostPort (util.js:1024:20)
    at Server.setupListenHandle [as _listen2] (net.js:1353:14)
    at listenInCluster (net.js:1394:12)
    at GetAddrInfoReqWrap.doListen [as callback] (net.js:1509:7)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:102:10)
  errno: 'EADDRINUSE',
  code: 'EADDRINUSE',
  syscall: 'listen',
  address: '127.0.0.1',
  port: 8080 }
ℹ ｢hot｣: webpack: Compiling Done
```

webpack-serve is able to continue running and host files after this, which is commendable, but every restart also leaves yet another node exe running and eating memory in the background: https://i.imgur.com/HMVXwbE.jpg

### How Do We Reproduce?
Start webpack-serve on Windows 10 (using Git Bash included with the official Git client), hit Ctrl+C to kill, then start again and see if you get the 'port in use' error.

**Comment by shellscape** [comment]
Thanks for the issue 🍺  Unfortunately we're already using what's available to us to determine if the process should end, and we've got the right sequence of handlers in place to end the server. You can view that code here: https://github.com/webpack-contrib/webpack-serve/blob/8d160d8e1f6ee75935d836688d6822c053af3297/index.js#L65-L73

Windows unfortunately still sucks at the console/terminal. webpack-dev-server would see an issue on this reported every few months fairly regularly. If you can figure out a fix for your env on windows, we'd happily welcome a Pull Request.

**Comment by shellscape** [comment]
We're closing this issue as stale as it's more than 30 days without activity, and without an associated Pull Request. Please feel free to continue discussion. We'll reopen this issue if anything actionable is posted.

**Comment by shellscape** [comment]
This was resolved in #110 

/cc @ulivz

**Comment by ulivz** [comment]
Thanks a lot ! @shellscape
